IDE/MDK5-ARM/Projects/wolfSSL-Full/RTE/wolfSSL/user_settings.h

```diff
@@ -0,0 +1,516 @@
+/* user_settings.h
+ *
+ * Copyright (C) 2006-2017 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#define NO_MAIN_DRIVER
+#define BENCH_EMBEDDED
+#define NO_DEV_RANDOM
+#define WOLFSSL_USER_CURRTIME
+#define SIZEOF_LONG_LONG 8
+#define NO_WRITEV
+#define NO_DEV_RANDOM
+
+#define TFM_TIMING_RESISTANT
+#define ECC_TIMING_RESISTANT
+#define WC_RSA_BLINDING
+
+#define WOLFSSL_USER_CURRTIME /* for benchmark */
+#define WOLFSSL_CURRTIME_OSTICK /* use OS tich for current_time */
+#define WOLFSSL_GMTIME
+#define NO_MULTIBYTE_PRINT
+
+// <<< Use Configuration Wizard in Context Menu >>>
```

Comment:
Is this auto generated? needed? We try to avoid single line comment "//" style.

Reply:
They are a part of MDK configuration wizard syntax.